src/utils/vcs.rs

```diff
@@ -234,6 +234,24 @@ pub fn git_repo_remote_url(
         .ok_or_else(|| git2::Error::from_str("No remote URL found"))
 }
 
+#[cfg(feature = "unstable-mobile-app")]
+pub fn git_repo_head_ref(repo: &git2::Repository) -> Result<String, git2::Error> {
```

Comment:
We should just use the normal `anyhow::Result` here

```suggestion
pub fn git_repo_head_ref(repo: &git2::Repository) -> Result<String> {
```